CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+## 0.2.0 (2023-02-22)
+
+- Added support for named constants (scalars, vectors, and matrices).
+- Added support for intermediate variables in `boundary_constraints` and `transition_constraints` sections (scalars, vectors, and matrices).
+- Added support for binding identifier names to groups of trace columns (in addition to single columns, which were already supported).
+- Added the `$main` and `$aux` built-ins for accessing columns in the execution trace by index.
+- [BREAKING] Replaced the `$rand` built-in for accessing random values with an explicit `random_values` declaration section, enabling use of a custom identifier for the random values array.
+- Added support for binding identifiers to specific random values.
+- Made significant changes to the IR, including:
```

Comment:
Should we say "IR structure" or maybe "IR design" here instead?

Reply:
I think I'd leave it as it is - we've made changes to both the design and the structure, as well as the interface used for codegen, and "IR" encompasses all of that.